### PR TITLE
yorkie 0.2.6 (new formula)

### PR DESCRIPTION
**Pull request by hackerwins**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/yorkie.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    run_type :immediate
+    keep_alive true
+    working_dir var
+  end
+
+  test do
+    yorkie_pid = fork do
+      exec bin/"yorkie",
+        "server"
+    end
```
Comment (SMillerDev) [inline Formula/yorkie.rb:30]:
```suggestion
    yorkie_pid = fork do
      exec bin/"yorkie", "server"
    end
```

File: Formula/yorkie.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    # sleep to let yorkie get ready
+    sleep 3
+
+    shell_output("#{bin}/yorkie project ls")
+  ensure
+    # clean up the etcd process before we leave
```
Comment (SMillerDev) [inline Formula/yorkie.rb:36]:
```suggestion
    # clean up the process before we leave
```

File: Formula/yorkie.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  head "https://github.com/yorkie-team/yorkie.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install Dir[buildpath/"bin/*"]
+  end
+
+  plist_options manual: "yorkie"
```
Comment (SMillerDev) [inline Formula/yorkie.rb:17]: This won't do anything with a service block
```suggestion
```

File: Formula/yorkie.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+  service do
+    run [opt_bin/"yorkie"]
```
Comment (SMillerDev) [inline Formula/yorkie.rb:20]:
```suggestion
    run opt_bin/"yorkie"
```

**Comment by hackerwins** [comment]
@SMillerDev Thanks for the review. It's cleaner than before.

File: Formula/yorkie.rb [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  end
+
+  test do
+    yorkie_pid = fork do
+      exec bin/"yorkie", "server"
+    end
+    # sleep to let yorkie get ready
+    sleep 3
+
+    shell_output("#{bin}/yorkie project ls")
```
Comment (SMillerDev) [inline Formula/yorkie.rb:31]: Can we check some output here?

Comment (hackerwins) [reply Formula/yorkie.rb]: @SMillerDev Thanks for the good comments. I added output check logic.

File: Formula/yorkie.rb [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    tag:      "v0.2.6",
+    revision: "284ceddfaa6400ce6eaaffad4570393c729ebf81"
+  license "Apache-2.0"
+  head "https://github.com/yorkie-team/yorkie.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install Dir[buildpath/"bin/*"]
```
Comment (carlocab) [inline Formula/yorkie.rb:14]:
```suggestion
    prefix.install "bin"
```
Though, is there no way to specify the prefix in the Makefile?

Comment (hackerwins) [reply Formula/yorkie.rb:14]: Thanks, It looks good.

**Review by chenrui333 — APPROVED** [review]

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**